.changeset/violet-jars-visit.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sveltejs/kit': patch
+---
+
+[chore] upgrade to Undici 5.7.0
```

Comment:
In the title: [chore] upgrade to Undici 5.8.1, but it's just 5.7.0 :)

Reply:
i'm aware. i'm trying to figure out why the tests are failing